indico/modules/events/templates/admin/event_keywords.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'layout/admin_page.html' %}
+{% from 'forms/_form.html' import simple_form %}
+
+{% block title %}
+    {% trans %}Allowed event keywords{% endtrans %}
+{% endblock %}
+
+{% block description %}
+    {% trans %}
+        Define the list of allowed event keywords. An empty list implies any keyword is valid.
+    {% endtrans %}
+{% endblock %}
+
+{% block content %}
+    {{ simple_form(form, back_button=false, disabled_until_change=False) }}
+
+    <script>
+        domReady.then(function () {
```

Comment:
```suggestion
        domReady.then(() => {
```

Arrow functions are nicer :)

Reply:
Changed.